config/config.go

```diff
@@ -81,26 +85,28 @@ var (
 // DefaultConfig returns planetmint's default configuration.
 func DefaultConfig() *Config {
 	return &Config{
-		AssetRegistryScheme: "https",
-		AssetRegistryDomain: "testnet-assets.rddl.io",
-		AssetRegistryPath:   "register_asset",
-		TokenDenom:          "plmnt",
-		FeeDenom:            "plmnt",
-		StagedDenom:         "stagedcrddl",
-		ClaimDenom:          "crddl",
-		ConfigRootDir:       "",
-		PopEpochs:           24, // 24 CometBFT epochs of 5s equate 120s
-		RPCHost:             "localhost",
-		RPCPort:             18884,
-		RPCUser:             "user",
-		RPCPassword:         "password",
-		RPCScheme:           "http",
-		RPCWallet:           "rpcwallet",
-		ReissuanceAsset:     "7add40beb27df701e02ee85089c5bc0021bc813823fedb5f1dcb5debda7f3da9",
-		ValidatorAddress:    "plmnt1w5dww335zhh98pzv783hqre355ck3u4w4hjxcx",
-		DistributionAddrInv: "vjTyRN2G42Yq3T5TJBecHj1dF1xdhKF89hKV4HJN3uXxUbaVGVR76hAfVRQqQCovWaEpar7G5qBBprFG",
-		DistributionAddrDAO: "vjU8eMzU3JbUWZEpVANt2ePJuPWSPixgjiSj2jDMvkVVQQi2DDnZuBRVX4Ygt5YGBf5zvTWCr1ntdqYH",
-		DistributionAddrPop: "vjTvXCFSReRsZ7grdsAreRR12KuKpDw8idueQJK9Yh1BYS7ggAqgvCxCgwh13KGK6M52y37HUmvr4GdD",
+		AssetRegistryScheme:       "https",
+		AssetRegistryDomain:       "testnet-assets.rddl.io",
+		AssetRegistryPath:         "register_asset",
+		TokenDenom:                "plmnt",
+		FeeDenom:                  "plmnt",
+		StagedDenom:               "stagedcrddl",
+		ClaimDenom:                "crddl",
+		ConfigRootDir:             "",
+		PopEpochs:                 24, // 24 CometBFT epochs of 5s equate 120s
+		RPCHost:                   "localhost",
+		RPCPort:                   18884,
+		RPCUser:                   "user",
+		RPCPassword:               "password",
+		RPCScheme:                 "http",
+		RPCWallet:                 "rpcwallet",
+		ReissuanceAsset:           "7add40beb27df701e02ee85089c5bc0021bc813823fedb5f1dcb5debda7f3da9",
+		ValidatorAddress:          "plmnt1w5dww335zhh98pzv783hqre355ck3u4w4hjxcx",
+		DistributionAddrEarlyInv:  "tbd",
```

Comment:
Let's create one

Reply:
I used the old one for the default values on testnet